src/components/NavBar/NavBar.css

```diff
@@ -1,11 +1,14 @@
 nav {    
     background-color:rgb(94, 93, 156);    
     color: #ffffff;    
-    padding: 10px;     
+    padding: 10px; 
+    
 }
 .logo-navbar{
     background-color: lawngreen;
-    width: 250px;
+    width: 500px;
```

Comment:
fixed sizes usually are not the best options .. can you use `em`s or percentages ?

Reply:
Thanks for code review Ivan I changed it :)